modules/apps/object/object-admin-rest-impl/rest-openapi.yaml

```diff
@@ -349,6 +351,8 @@ components:
                 parameterObjectFieldId:
                     format: int64
                     type: integer
+                parameterObjectFieldName:
+                    type: string
```

Comment:
@p-albuquerque why `objectFieldName` and not the `objectFieldExternalReferenceCode`?

Reply:
Because it is requested in the [ticket](https://issues.liferay.com/browse/LPS-168214)

Reply:
thanks!